### PR TITLE
[IMPROVE ] : sort votes by count

### PR DESCRIPTION
**Pull request by RahulNavneeth**
# Description:

This PR adds a new feature to sort the votes by count isolated, as requested by @murtaza98.

## Screenshot / Video

https://user-images.githubusercontent.com/50259284/224066152-fe0b499b-14d2-454d-8fed-b1bc7c97a1a7.mov

### CODE REVIEW

File: src/lib/storeVote.ts [hunk 1]
```diff
@@ -26,6 +27,16 @@ export async function storeVote(poll: IPoll, voteIndex: number, { id, username,
...
 
+    let data: Array<[string, IVoter]> = [];
+    for (let i = 0, n = poll.votes.length; i < n; i++) {
+        data.push([poll.options[i], poll.votes[i]]);
+    }
+
+    data = data.sort((j, i) => i[1].quantity - j[1].quantity);
+
+    poll.votes = data.map((i) => i[1])
+    poll.options = data.map((i) => i[0])
```
[the remark this answers is not in the document]
Comment (RahulNavneeth) [reply src/lib/storeVote.ts]: Agreed and implemented sir 👍

**Comment by RahulNavneeth** [comment]
Hi @murtaza98 , Can you kindly review the changes made. Thanks 😄.

**Review by murtaza98 — APPROVED** [review]

**Comment by murtaza98** [comment]
Approved and merged :heavy_check_mark: Congrats, that was indeed a good improvement :clap:

**Comment by RahulNavneeth** [comment]
> Approved and merged ✔️ Congrats, that was indeed a good improvement 👏

Thanks you sir 😄

**Pull request merged**